Automattic/wp-calypso
# My Sites Misc: Remove problematic "blacklist"/"whitelist" language #43395

#### Changes proposed in this Pull Request

* Remove problematic language from documentation, my-sites/themes and my-sites/marketing

#### Testing instructions

* Ensure unit tests pass
* Ensure search welcome card works as expected

Addresses part of #43395
